feat(ins): add how-to migrate to sbs (#4868)
scaleway/docs-content: pages/instances/how-to/migrate-local-storage-to-sbs.mdx

---
meta:
  title: How to migrate Instance Local Storage to Block Storage volumes
  description: This page explains how to migrate Instance Local Storage to Block Storage volumes.
  h1: How to migrate Instance Local Storage to Block Storage volumes
  paragraph: This page explains how to migrate Instance Local Storage to Block Storage volumes.

> tags: instance migration local sroage block storage sbs

Reviewer: ```suggestion
tags: instance migration local storage block storage sbs
```

dates:
  validation: 2025-04-22

>   posted: 2024-04-22

Reviewer: The year should be 2024?

Author: I'm living in past.

categories:
  - compute
---

This tutorial guides you through the process of migrating your Instance's Local Storage (`l_ssd`) to Scaleway Block Storage (SBS). 

> By following these steps, you will be able to export your local storage and restart your Instance with SBS.

Reviewer 2: ```suggestion
By following these steps, you will be able to export your Local Storage and restart your Instance with SBS.
```

1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
2. Click the Instance you want to snapshot.
3. Click the **Storage** tab to display the storage options of your Instance.
4. Click <Icon name="more" /> > **Snapshot** next to the volume you want to snapsot.
5. A pop-up displays. Choose a **name** for the snapshot, or use the default suggested name.
6. Click **Create snapshot** to finish. You are redirected to the **Snapshots** tab in the **Local Storage** section of the Scaleway console, where your new snapshot now appears in the list.

## Exporting the snapshot

Once you have created the snapshot, you need to export it. This will allow you to import the snapshot into a new SBS volume.

1. Click the [snapshots](https://console.scaleway.com/local-storage/snapshots) tab in the **Local Storage** section of the Scaleway console.
2. A list of your Local Storage snapshots displays. Click <Icon name="more" /> > **Copy to bucket**.

> 3. A pop-up displays. Enter a name for the QCOW2 export of your snapshot and select the Object Storage bucket to export your snapshot into from the drop-down list.

Reviewer: ```suggestion
3. A pop-up displays. Enter a name for the QCOW2 export of your snapshot and, from the drop-down list, select the Object Storage bucket to export your snapshot to.
```

2. Click **Attach an Instance** under the **Linked Instance** section.
3. Select the Instance to which you want to attach your volume from the drop-down.
4. Click **Attach volume to Instance**.

<Message type="tip">
  Alternatively you can [create a new Instance](/instances/how-to/create-an-instance/) and use the SBS volume as boot volume for your Instance.
</Message>

## Restarting your Instance

After attaching the new SBS volume, you need to restart your instance. This will ensure that your Instance is using the new SBS volume as its primary storage.

1. Click **Instances** in the **Compute** section of the side menu. The [Instances page](https://console.scaleway.com/instance/servers) displays.
2. Click the Instance you want to reboot.
3. Click **Actions** > **Reboot** in the top right corner of the Instance overview page.
4. A pop-up displays to confirm the Instance reboot. Click **Reboot Instance**.

## Conclusion

By following these steps, you have successfully migrated your local storage (`l_ssd`) to Scaleway Block Storage (SBS).
Your Instance is now up and running using SBS, and you can benefit from the advantages of Scaleway's advanced Block Storage solution.

> For furher information, refer to [How to migrate existing volumes and snapshots to Scaleway's new Block Storage management](/instances/how-to/migrate-volumes-snapshots-to-sbs/).

Reviewer: ```suggestion
For further information, refer to [How to migrate existing volumes and snapshots to Scaleway's new Block Storage management](/instances/how-to/migrate-volumes-snapshots-to-sbs/).
```

<Message type="note">

>   This tutorial does not cover specific modifications done by the user in the Instance that would break the mounting system.

Reviewer 2: ```suggestion
  This tutorial does not cover specific modifications made by the user in the Instance that would break the mounting system.
```

>   If you have made any modifications, ensure that they are compatible with SBS before proceeding with the migration.

Reviewer: ```suggestion
  If you have made any modifications, ensure they are SBS-compatible before migrating.
```
